gradle/verification-metadata.xml

```diff
@@ -2112,17 +2112,29 @@
             <sha256 value="0bdd11c226ff836057f753f8426b4f77efc5fdba96f55692eceea82656c93ac0" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.gradle" name="common-custom-user-data-gradle-plugin" version="1.6.5">
+         <artifact name="common-custom-user-data-gradle-plugin-1.6.5.jar">
+            <md5 value="f0351444c899aeb94084d21cb773fe53" origin="Generated by Gradle"/>
+            <sha256 value="67364c77f47ba09815417f3157aa85d506f3710b2f058d46822879cf87646dde" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.6.3">
          <artifact name="gradle-enterprise-gradle-plugin-3.6.3.jar">
             <md5 value="680b8913e65b28a1fe378e277d81877f" origin="https://plugins.gradle.org/m2/com/gradle/gradle-enterprise-gradle-plugin/3.6.2/"/>
             <sha1 value="415ea1284e690c0573376eccf8e7d7542499a3ef" origin="https://plugins.gradle.org/m2/com/gradle/gradle-enterprise-gradle-plugin/3.6.2/"/>
             <sha256 value="23148e8925cf140a6d5fe6c0b58368580ca1b93400d73031286a5b14b356d9b0" origin="Generated by Gradle"/>
          </artifact>
       </component>
-      <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.8.1">
-         <artifact name="gradle-enterprise-gradle-plugin-3.8.1.jar">
-            <md5 value="4bb7b6821ebedbaf98db8ad50ab7c709" origin="Generated by Gradle"/>
-            <sha256 value="244ce3fb2d37bed404c556691508ca828eef2539dcb671e7215b90c0156509fa" origin="Generated by Gradle"/>
+       <component group="com.gradle" name="gradle-enterprise-gradle-plugin" version="3.8.1">
```

Comment:
can we remove this?

Reply:
My first attempt was without this block and the build failed as missing this, I'll give it another try

Reply:
oups, it did it again
https://github.com/jprinet/gradle-enterprise-oss-projects/runs/5885654467?check_suite_focus=true#step:5:151

Do one need to trigger the build twice to be able to get rid of old version?

Reply:
Got it, the automated validation scripts use version 3.8.1
https://ge.solutions-team.gradle.com/s/kbc552lnvsyqq/build-dependencies?dependencies=gradle-e&expandAll

Reply:
So is this only here because of the build validation scripts? is it needed for the actual build?

Reply:
I'm currently running the scripts and this is the reason for having this 3.8.1, but the final PR won't contain it 👍